src/module.c

```diff
@@ -1447,6 +1447,63 @@ int populateArgsStructure(struct redisCommandArg *args) {
     return count;
 }
 
+int moduleVerifyName(const char *name) {
+    if (strlen(name) == 0) {
+        return REDISMODULE_ERR;
+    }
+
+    for (size_t i = 0 ; i < strlen(name) ; ++i) {
+        char curr_char = name[i];
+        if ((curr_char >= 'a' && curr_char <= 'z') ||
+            (curr_char >= 'A' && curr_char <= 'Z') ||
+            (curr_char >= '0' && curr_char <= '9') ||
+            (curr_char == '_') || (curr_char == '-'))
+        {
+            continue;
+        }
+        return C_ERR;
+    }
+    return C_OK;
+}
+
+/* RedisModule_AddACLCategory can be used to add new ACL command categories.
+ * This function can only be called during the RedisModule_OnLoad function.
```

Comment:
```suggestion
/* RedisModule_AddACLCategory can be used to add new ACL command categories. Category names
 * can only contain alphanumeric characters, underscores, or dashes. Categories can only be added
 * during the RedisModule_OnLoad function. Once a category has been added, it can not be removed. 
 * Any module can register a command to any added categories using RedisModule_SetCommandACLCategories.
```